Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Adressed #1104 holding types

## Changelog:
- Fixed issues around the world where rulers had holdings of the wrong type in their possession.


# How to test:
Start any bookmark, pick a few provinces to make sure rulers don't have holdings of the wrong type.  Provinces that had issues mentioned here: #1104.
